config/environments/development.rb.SAMPLE

```diff
@@ -39,7 +39,7 @@ Foodsoft::Application.configure do
   # Mailcatcher config, start mailcatcher from console with 'mailcatcher'
   # Mailcatcher can be installed by gem install mailcatcher
   config.action_mailer.delivery_method = :smtp
-  config.action_mailer.smtp_settings = { address: "localhost", port: 1025 }
+  config.action_mailer.smtp_settings = { address: "mailcatcher", port: 1025 }
```

Comment:
Quickly looking over the changes again, should we get this from an environment variable with a `localhost` default, so that it works both with and without Docker?